.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+python:
+ - 3.6
+
+cache: pip
+
+addons:
+  apt:
+    packages:
+      - libssl-dev
+      - python3-dev
+
+before_script:
```

Comment:
travis automatically installs `requirements.txt` ; trick I use iscat  test-requirements.txt

Reply:
Fancy, I like it!